articles/connections/social/auth0-oidc.md

```diff
@@ -101,26 +107,17 @@ Once the user is authenticated, the resulting profile will contain the [Auth0 No
 
 Note that the generated `user_id` has the following format:
 
-`auth0-oidc|YOUR_AUTH0_CONNECTION_NAME|THE_CHILD_AUTH0_CONNECTION|THE_CHILD_USER_ID`
+`auth0-oidc|YOUR_AUTH0_CONNECTION_NAME|THE_OIDC_PROVIDER_AUTH0_CONNECTION|THE_OIDC_PROVIDER_USER_ID`
 
-The `access_token` is the JWT of the user in the child Auth0 connection. If you decode it, you will see all the properties that were requested in the `scope` of the auth0-oidc connection. For example, for `scope=openid identities` will return:
+The `access_token` is the JWT of the user in the OIDC Provider Auth0 connection. If you decode it, you will see all the properties that were requested in the `scope` of the auth0-oidc connection. For example, for `scope=openid email` will return:
```

Comment:
Access token (if it is JWT when audience is used) and the id token should have the `iss` of `Relaying Party` not the `OIDC Provider`. This is a bug inherited from the old version of the doc. May be we can also correct them.

Reply:
Is there even a reason for this section to discuss decoding an access token? Why can't it just say that the requested claims will be included?

Reply:
I'm not even sure why the server is putting an `id_token` in a field name `access_token`. Might be a better idea to remove the mention completely.